src/main/java/com/akkus/RestDemoApplication.java

```diff
@@ -0,0 +1,37 @@
+package com.akkus;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import static org.slf4j.LoggerFactory.getLogger;
+import org.springframework.boot.SpringApplication;
+import static org.springframework.boot.SpringApplication.run;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+/**
+ *
+ * @author akkus
+ */
+@SpringBootApplication
+@EnableScheduling
+@EnableAutoConfiguration
+@ComponentScan("com.akkus.*")
+public class RestDemoApplication {
+
+    /**
+     *
+     */
+    public final static Logger logger = getLogger(RestDemoApplication.class);
```

Comment:
What is the purpose of this line?

Reply:
Actually I used one of my old project as template.I have missed to delete this part.